lib/core/plugins/context.js

```diff
@@ -282,15 +282,16 @@ function curryTrigger (kuzzle, pluginName) {
   /**
    * @this   {Kuzzle}
    * @param  {String} eventName The name of the custom event to trigger.
-   * @param  {Object} payload   The payload of the event.
+   * @param  {Any} payload      The payload of the event.
    */
   return function trigger (eventName, payload) {
     if (eventName.indexOf(':') !== -1) {
```

Comment:
I can't recall why we have such a limitation. And I wonder if that doesn't make triggering events impractical... Or even usable. What do you think?

Reply:
Yes, I removed it :+1: